clang/lib/Headers/intrin0.h

```diff
@@ -44,7 +44,7 @@ unsigned char _InterlockedCompareExchange128_rel(__int64 volatile *_Destination,
                                                  __int64 *_ComparandResult);
 #endif
 
-#ifdef __x86_64__
+#ifdef __x86_64__ && !defined(__arm64ec__)
```

Comment:
Hello, we found compile errors on this line:
```
(47,19): error: extra tokens at end of #ifdef directive [-Werror,-Wextra-tokens]
```
Seems like this line forgot to change #ifdef when changed into expression

Reply:
@FreddyLeaf Curiosity how are you running clang. Since `intrin0.h` is treated as a system header warnings should be suppressed.

I couldn't repro the warning in my local testing due to `intrin0.h` being found via a system include path.

Reply:
Sorry I don't have the reproducer, either. The scene supposed to be building V-Ray with MSbuild on Windows.